CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+- Add support for payload length tag in the signature header. Newly built RPMs will have this set automatically.
```

Comment:
Update the changelog entry also, and put it under a `### Fixed` or `### Changed` section heading. You can mention that this is aligning with RPMv4, which suggests that the tag should be present.